VK_App/VK_App/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,65 @@
+// NetworkService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// NetworkService - класс для работы с сетью
+final class NetworkService {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let baseUrl = "https://api.vk.com"
+        static let path = "/method"
+        static let userID = "user_id"
+        static let accessToken = "access_token"
+        static let extended = "extended"
+        static let ownerId = "owner_id"
+        static let vKey = "v"
+        static let qKey = "q"
+    }
+
+    enum Methods: String {
+        case groups = "/groups.get"
+        case searchGroups = "/groups.search"
+        case friendsList = "/friends.get"
+        case photo = "/photos.getAll"
+    }
+
+    // MARK: - Public Properties
+
+    static let instance = NetworkService()
+
+    var token: String?
+    var userId: String?
+
+    // MARK: - Private Initializers
+
+    private init() {}
+
+    // MARK: - Public Method
+
+    func getResponse(method: Methods) {
```

Comment:
```suggestion
    func fetchResponse(method: Methods) {
```
у нас могут быть get, post  запросы,
лучше писать  fetch